var/spack/repos/builtin/packages/mumps/package.py

```diff
@@ -19,6 +19,8 @@ class Mumps(Package):
     variant('float', default=True, description='Activate the compilation of smumps')
     variant('complex', default=True, description='Activate the compilation of cmumps and/or zmumps')
     variant('idx64', default=False, description='Use int64_t/integer*8 as default index type')
+    variant('tests', default=False, description='Run small MPI unit tests')
```

Comment:
tests won't change what get installed, yet they add another flavor to the package. I don'think it's the best way to conditionally enable part of the code.